crates/net/network/src/session/active.rs

```diff
@@ -51,6 +51,10 @@ const SAMPLE_IMPACT: f64 = 0.1;
 /// Amount of RTTs before timeout
 const TIMEOUT_SCALING: u32 = 3;
 
+/// The type of the underlying network connection.
```

Comment:
alternatively, we could box the snap::Encoder, but since we should box some of it, we can just box the entire thing...

I'd like to rename this though, like PeerConnection